modules/openapi-generator/src/test/resources/2_0/petstore-with-fake-endpoints-models-for-testing.yaml

```diff
@@ -1041,6 +1041,35 @@ paths:
         - petstore_auth:
             - 'write:pets'
             - 'read:pets'
+  /fake/create_xml_item:
```

Comment:
@spacether I think you will need to add 2 more spaces to indent the new fake endpoint so that it will be grouped into the previous endpoint `POST /fake/{petId}/uploadImageWithRequiredFile`

Reply:
@wing328 why you think I need to further indent it?
The code I added creates a new route /fake/create_xml_item which has a post method.
Looking at higher up examples, my indenting is consistent with the definition of other routes.
Indent 2 spaces for the route
Indent 4 spaces for the method

This test must be a post route so we can submit a body.
Are you suggesting that I move the XmlItem body into the existing uploadImageWithRequiredFile post route?
I'd prefer to keep them separate as they test different features.

Did you see my comment above?
Where is the rest-assured java generator file?

Reply:
> Why do you think I need to further indent it?

`/fake/create_xml_item:` is put in the DefaultAPI class (a new class) so I don't think it's grouped under the same tag in the previous endpoint. Please change the indention to give it a try. (ideally I want to avoid another new class for a new fake endpoint)

Reply:
It was the missing tag that made the route generate a new defaultapi file. Adding the tag fixed it.